package.json

```diff
@@ -6,12 +6,12 @@
   "scripts": {
     "build": "run-s build:*",
     "build:clean": "if test -d ./lib; then rm -r ./lib; fi",
-    "build:babel": "babel src/ --out-dir lib/ --minified",
-    "watch": "npm run build -- --watch",
+    "build:babel": "babel src/ --out-dir lib/",
+    "pretest": "npm run build",
```

Comment:
I changed the test target from `src/` to `lib/` so I built each time before test.
Because the amount of code is not large, the build will be completed soon.